variables.tf

```diff
@@ -215,42 +216,51 @@ variable "etcdLBShape" {
 
 variable "etcd_lb_enabled" {
   description = "enable/disable the etcd load balancer. true: use the etcd load balancer ip. false:use a list of etcd instance ips."
-  default = "true"
+  default     = true
 }
 
 variable "k8sMasterLBShape" {
   default = "100Mbps"
 }
 
+variable "master_oci_lb_enabled" {
+  description = "enable/disable the k8s master load balancer. true: use the k8s master load balancer ip. false: use a nginx reverse proxy for k8s masters."
```

Comment:
From https://www.terraform.io/docs/configuration/variables.html#booleans:

> It is recommended for now to specify boolean values for variables as the strings "true" and "false", to avoid some caveats in the conversion process. A future version of Terraform will properly support boolean values and so relying on the current behavior could result in backwards-incompatibilities at that time.

Here's why:

```
grep master_oci_lb_enabled terraform.tfvars
master_oci_lb_enabled = "false"
```

```
terraform plan

Error: Error running plan: 2 error(s) occurred:

* module.k8smaster-public-lb.oci_load_balancer.lb-k8smaster: cannot parse "false" as an integer
* module.etcd-lb.oci_load_balancer.lb-etcd: cannot parse "false" as an integer
```